app/src/test/java/org/simple/clinic/registration/location/RegistrationLocationPermissionScreenControllerTest.kt

```diff
@@ -31,9 +33,10 @@ class RegistrationLocationPermissionScreenControllerTest {
   @Test
   fun `when location permission is received then facility selection screen should be opened`() {
     setupController()
-    uiEvents.onNext(RequestLocationPermission(permission = Just(GRANTED)))
+    uiEvents.onNext(RequestLocationPermission(permission = Optional.of(GRANTED)))
```

Comment:
Does `of()` in `Optional.of(...)` mean this will return an optional with non-null value?

Reply:
Yep